### DISCUSSION TITLE
The mesh shader is red and nothing shows up in the viewport.

### DISCUSSION THREAD

**Discussion by rosstheblenderanimator** [bnpr/Malt#521] [Q&A]
![Screenshot 2023-09-28 145717](https://github.com/bnpr/Malt/assets/146353254/e9a583e5-5fc3-46da-a406-20405cc91214)
Does anybody know why this might be happening?
![Screenshot 2023-09-27 183831](https://github.com/bnpr/Malt/assets/146353254/19337027-e690-45e2-98c0-1a1ba76243b0)

**Comment by rosstheblenderanimator** [comment]
Also the tiny black window does not show up.

**Comment by eballeste** [comment]
hey, you should report this as a possible bug, you'll get to fill in more details that would help Miguel troubleshoot your issue a lot better

it also has a link to the readme file so you can make sure that you have everything setup correctly before reporting the issue

https://github.com/bnpr/Malt/issues/new?assignees=&labels=&projects=&template=bug_report.yml
